### DISCUSSION TITLE
Also Delete From CoE

### DISCUSSION THREAD

**Discussion by mschuess** [microsoft/powerapps-tools#953] [Q&A] [answered]
When the Admin | Sync Template v3 (Check Deleted) flow is run, this denotes whether you want the items deleted from CoE (Yes, which is the default)

Can you provide just a bit more detail on what it means to set this environment variable to yes or no?

**Comment by manuelap-msft** [comment, marked as answer]
Hello,

Apologies for the delay, our team only receives notifications for the Issues tab here and we didn't realise people also raised questions under Discussions. We'll keep an eye on it now! 

The CoE stores meta data about your inventory in Dataverse tables. When an app, environment or flow is deleted from your tenant, you can either also delete it from the CoE table (remove the record from the table) or highlight it as deleted in the table (set a flag to say its deleted but keep the record). That's what this variable let's you decide!
